maxim/layers.py

```diff
@@ -76,28 +100,60 @@ def get_config(self):
 
 
 @tf.keras.utils.register_keras_serializable("maxim")
-class Resizing(layers.Layer):
-    def __init__(self, height, width, antialias=True, method="bilinear", **kwargs):
+class ResizingDown(tf.keras.layers.Layer):
+    def __init__(self, ratio: float, method="bilinear", antialias=True, **kwargs):
         super().__init__(**kwargs)
-        self.height = height
-        self.width = width
-        self.antialias = antialias
+        self.ratio = ratio
         self.method = method
+        self.antialias = antialias
 
-    def call(self, x):
-        return tf.image.resize(
-            x,
-            size=(self.height, self.width),
+    def __call__(self, img):
```

Comment:
Prefer `call()` or is there anything I am missing out on?

Reply:
No, fixing...